src/content/architecture/_index.en.md

```diff
@@ -1,7 +1,21 @@
----
-title: "Architecture"
-date: 2020-02-19T21:00:30+01:00
-pre: "<b>3. </b>"
-weight: 15
----
++++
+title = "Architecture"
+date = 2020-02-19T21:00:30+01:00
+pre = "<b>3. </b>"
```

Comment:
Thanks for incorporating other comments.

I see that index for Architecture page is 3 and is shown before other pages. Please take a look.
![9](https://user-images.githubusercontent.com/6574656/81494974-6c862080-92ca-11ea-87a0-b42d32d9e345.jpg)

Reply:
Setting weight = 10 on the Architecture page should fix this issue.

Reply:
I set the weights appropriately so they're ordered properly. I don't know where the numbers are coming from - it is not in the index files where the weight is. This rendering tool is rather bizarre if you ask me 😄 Since this is a pre-existing issue not related to this PR, can we merge this and investigate the numbering later?

Reply:
Oh wait - I found it - it's "pre: "<b>3. </b>" - duh 😄

Reply:
It was the architecture weight 😄 I was sure I had changed those correctly before. Fixed now.